packages/office-ui-fabric-react/src/Utilities.ts

```diff
@@ -1 +1,2 @@
 export * from '@uifabric/utilities/lib/index';
+export * from './utilities/selectableOption';
```

Comment:
E.g. in components/DetailsList/index.ts, you can see we have a number of exports:

```ts
export * from '../../utilities/selection/index';
export * from '../GroupedList/GroupedList.Props';
export * from './DetailsList';
export * from './DetailsList.Props';
export * from './DetailsRow';
export * from './DetailsRowCheck';
```